src/cljs/harja/tiedot/hallinta/tyokalut/talvihoitoreitti_tyokalu.cljs

```diff
@@ -0,0 +1,251 @@
+(ns harja.tiedot.hallinta.tyokalut.talvihoitoreitti-tyokalu
+  "Toteumatyokalun ui controlleri."
+  (:require [reagent.core :refer [atom] :as reagent]
+            [tuck.core :as tuck]
+            [harja.pvm :as pvm]
+            [harja.tyokalut.tuck :as tuck-apurit]
+            [harja.tyokalut.yleiset :as yleiset]
+            [cljs.core.async :refer [<! >! chan close!]]
+            [cljs-http.client :as http]
+            [harja.asiakas.kommunikaatio :as k]
+            [harja.tiedot.navigaatio :as nav]
+            [harja.ui.viesti :as viesti]
+            [harja.tiedot.istunto :as istunto])
+  (:require-macros [reagent.ratom :refer [reaction]]
+                   [cljs.core.async.macros :refer [go]]))
+
+(def +mahdolliset-urakat+
+  [
+   {:id 34 :nimi "Ivalon MHU testiurakka (uusi) (aseta sopimusid 19)"}
+   {:id 35 :nimi "Oulun MHU 2019-2024 (Aseta sopimusid 42)"}
+   ])
```

Comment:
Näitä voisi formatoida oikein clojureksi, eli ei jätetä noita roikkuvia sulkeita